I get this error when testing this:

```
<testsuite name="kuka_ros2_control_support.ros2_control_support.launch_tests[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" tests="5" failures="0" errors="1" skipped="0" time="4.117"><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_joint_state_msg_retrieval[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="1.006" /><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_node_started[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="0.002" /><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_tcp_tf[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="0.065" /><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_trajectory_following[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="3.001"><error message="Traceback (most recent call last):&#10;  File &quot;/home/deniss/workspace/ros_industrial/kuka_robots_tests/src/kuka_experimental/kuka_ros2_control_support/test/ros2_control_support.py&quot;, line 288, in test_trajectory_following&#10;    joint_states = get_joint_states_from_trajectory_state()&#10;  File &quot;/home/deniss/workspace/ros_industrial/kuka_robots_tests/src/kuka_experimental/kuka_ros2_control_support/test/ros2_control_support.py&quot;, line 285, in get_joint_states_from_trajectory_state&#10;    return dict(zip(msg.joint_names, msg.actual.positions))&#10;AttributeError: 'NoneType' object has no attribute 'joint_names'&#10;" /></testcase><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_trajectory_state_retrieval[description_package=kuka_kr10_support description_macro_file=kr10r1100sixx_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="0.042" /></testsuite><testsuite name="kuka_ros2_control_support.ros2_control_support.launch_tests[description_package=kuka_kr10_support description_macro_file=kr10r1420_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" tests="5" failures="0" errors="0" skipped="0" time="3.268"><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_joint_state_msg_retrieval[description_package=kuka_kr10_support description_macro_file=kr10r1420_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="0.596" /><testcase classname="kuka_ros2_control_support.RobotDriverTest" name="test_node_started[description_package=kuka_kr10_support description_macro_file=kr10r1420_macro.xacro controllers_file=kuka_6dof_controllers.yaml initial_positions_file=initial_positions.yaml]" time="0.002" />
```

It seems there is still some `NoneType` issue or for the given robot type the setup is not correct.

kuka_robots_tests/src/kuka_experimental/kuka_ros2_control_support/test/ros2_control_support.py&quot;, line 285, in get_joint_states_from_trajectory_state&#10;    return dict(zip(msg.joint_names, msg.actual.positions))&#10;AttributeError: 'NoneType' object has no attribute 'joint_names'&#10;" />

--- a/kuka_ros2_control_support/test/ros2_control_support.py
+++ b/kuka_ros2_control_support/test/ros2_control_support.py
@@ -224,6 +224,7 @@ class RobotDriverTest(unittest.TestCase):
         cls.node_name = "ros2_control_support_test"
         cls.node = Node(cls.node_name)
         cls.executor = rclpy.executors.SingleThreadedExecutor()
+        cls.executor.add_node(cls.node)
         cls.traj_state_topic = "/position_trajectory_controller/state"
         cls.joint_trajectory_topic = "/position_trajectory_controller/joint_trajectory"
         cls.trajectory_publisher = cls.node.create_publisher(

